src/home_robot/home_robot/navigation_planner/discrete_planner.py

```diff
@@ -390,7 +388,9 @@ def _get_short_term_goal(
 
         # Select closest point on goal map for visualization
         # TODO How to do this without the overhead of creating another FMM planner?
-        vis_planner = FMMPlanner(traversible)
+        traversible_ = traversible.copy()
+        traversible_[dilated_goal_map == 1] = 1
```

Comment:
why are we doing this?

Reply:
So here we are computing distance from the agent's current position to all traversable points. If the goal is on an obstacle, it can get missed. So we dilate the goal and mark the region as traversable. Let me share a visual example as well

Reply:
![traversible_masked](https://user-images.githubusercontent.com/28150617/231852092-258ffbe1-ccd4-408b-a4d3-2b64991c6756.png)
 The yellow is the traversable map, purple is goal. For all goal points, `skfmm.distance` to agent will be high.

Reply:
we can just ignore "traversible" - https://github.com/facebookresearch/home-robot/blob/461bc549ee943a7628693a1331c387780f727af9/src/home_robot/home_robot/navigation_planner/discrete_planner.py#L504 this is what i was doing in the new PR

Reply:
Assumes that the closest goal is the one we're trying to manipulate, to be fair, but this is a baseline

Reply:
Yes, that will fix this. I will remove the discrete_planner changes from here. The other changes in the PR are necessary though.

Reply:
I think this will conflict with the new code in #134 - but we can merge it for now, I may want your help to make sure the two code paths are properly supported + switch to using the same config on hardware